<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Estilos comunes para varios controles */
QWidget {
    background-color: #deede3;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Clientes</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>871</width>
      <height>501</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ver, agregar o editar un cliente&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Estilos para QTabWidget */
QTabWidget {
    background-color: #deede3;
    border: 1px solid #cccccc;
    border-radius: 4px;
}

QTabWidget::tab-bar {
    left: 5px;
}

QTabWidget::tab {
    background-color: #deede3;
    border: 1px solid #1e362d;
    border-bottom-left-radius: 4px;
    border-bottom-right-radius: 4px;
    padding: 6px 12px;
}

QTabWidget::tab:selected {
    background-color: #c0dacb;
    border: 1px solid #1e362d;
    border-bottom: 1px solid #c0dacb;
}

QTabWidget::pane {
    background-color: #deede3;
    border: 1px solid #1e362d;
    border-top: none;
    border-bottom-left-radius: 4px;
    border-bottom-right-radius: 4px;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Listado</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>831</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid #1e362d;
    border-radius: 8px;
    padding: 10px;
    background-color: var(--jade-50);
}

QGroupBox:title {
    color: #1e362d;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding-left: 5px;
    padding-right: 5px;
    border: none;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>15</y>
         <width>120</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: #deede3;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QComboBox:hover {
    border: 1px solid #689f84;
}

QComboBox:focus {
    border: 1px solid #689f84;
    outline: none;
}

QComboBox::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
       </property>
       <item>
        <property name="text">
         <string>Nombre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Documento</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="txtBuscar">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>15</y>
         <width>120</width>
         <height>35</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Inserte el dato para buscar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #deede3;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #689f84;
}

QLineEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}

QLineEdit::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBuscar">
       <property name="geometry">
        <rect>
         <x>285</x>
         <y>15</y>
         <width>85</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #1e362d);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEliminar">
       <property name="geometry">
        <rect>
         <x>375</x>
         <y>15</y>
         <width>85</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #1e362d);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
       </property>
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnImprimir">
       <property name="geometry">
        <rect>
         <x>465</x>
         <y>15</y>
         <width>85</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #1e362d);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
       </property>
       <property name="text">
        <string>Imprimir</string>
       </property>
      </widget>
      <widget class="QTableView" name="tbDatos">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>771</width>
         <height>351</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView {
    background-color: #f2f7f4;
    border: 1px solid #cccccc;
    selection-background-color: #96bfaa;
    selection-color: #ffffff;
}

QTableView QHeaderView {
    background-color: #c0dacb;
    color: #333333;
    border: none;
}

QTableView QHeaderView::section {
    background-color: #deede3;
    color: #333333;
    padding: 6px;
    border: 1px solid #1e362d;
}

QTableView QHeaderView::section:checked {
    background-color: #96bfaa;
    color: #ffffff;
}

QTableView::item {
    padding: 6px;
}

QTableView::item:selected {
    background-color: #96bfaa;
    color: #ffffff;
}

QTableView::item:focus {
    background-color: #96bfaa;
    color: #ffffff;
    outline: none;
}</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mantenimiento</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>831</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid #1e362d;
    border-radius: 8px;
    padding: 10px;
    background-color: var(--jade-50);
}

QGroupBox:title {
    color: #1e362d;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding-left: 5px;
    padding-right: 5px;
    border: none;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>30</y>
         <width>55</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Codigo</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>70</y>
         <width>63</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>210</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Direccion</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGuardar">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>350</y>
         <width>86</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #1e362d);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEditar">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>350</y>
         <width>86</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #1e362d);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
       </property>
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSalir">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>350</y>
         <width>86</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #1e362d);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtCodigo">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>105</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #deede3;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #689f84;
}

QLineEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}

QLineEdit::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtNombre">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>105</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #deede3;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #689f84;
}

QLineEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}

QLineEdit::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtDireccion">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>200</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit {
    background-color: #deede3;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QTextEdit:hover {
    border: 1px solid #689f84;
}

QTextEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Apeliddos</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbTipoDocumento">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>105</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: #deede3;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QComboBox:hover {
    border: 1px solid #689f84;
}

QComboBox:focus {
    border: 1px solid #689f84;
    outline: none;
}

QComboBox::placeholder {
    color: #1e362d;
    font-weight: bold;
}
</string>
       </property>
       <item>
        <property name="text">
         <string>RNC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CEDULA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PASAPORTE</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>110</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Tipo Documento</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtNumDocumento">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>100</y>
         <width>181</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #deede3;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #689f84;
}

QLineEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}

QLineEdit::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>260</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Telefono</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtTelefono">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>250</y>
         <width>120</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #deede3;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #689f84;
}

QLineEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}

QLineEdit::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>310</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtEmail">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>300</y>
         <width>181</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #deede3;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #689f84;
}

QLineEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}

QLineEdit::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtApellidos">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>60</y>
         <width>161</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #deede3;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #689f84;
}

QLineEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}

QLineEdit::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Sexo</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbSexo">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>105</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: #deede3;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QComboBox:hover {
    border: 1px solid #689f84;
}

QComboBox:focus {
    border: 1px solid #689f84;
    outline: none;
}

QComboBox::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
       </property>
       <item>
        <property name="text">
         <string>M</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>F</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>160</y>
         <width>158</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Fecha de nacimiento</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="txtFechaNac">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>150</y>
         <width>105</width>
         <height>35</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Estilos para QDateEdit */
QDateEdit {
    background-color: #deede3;
    border: 1px solid #cccccc;
    border-radius: 4px;
    font-size: 14px;
    padding: 6px;
}

QDateEdit:hover {
    border: 1px solid #689f84;
}

QDateEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}



</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
